README.md

```diff
@@ -59,6 +59,12 @@ Semantic Kernel is available to explore AI and build apps with C# and Python:
           <a href="python/README.md">Using Semantic Kernel in Python</a>
         </div>
       </td>
+      <td>
+        <img align="left" width=52px src="https://raw.githubusercontent.com/devicons/devicon/master/icons/java/java-original.svg">
```

Comment:
This is an old Java logo.

This is the new one: https://en.wikipedia.org/wiki/Java_%28programming_language%29#/media/File:Java_programming_language_logo.svg

Reply:
@brunoborges says that one is not free to use: https://en.wikipedia.org/wiki/File:Java_programming_language_logo.svg

The one I'm using came from the same location as the other icons we are using.

Reply:
Please update to the new logo.